plane/src/typed_unix_socket/mod.rs

```diff
@@ -1,30 +1,17 @@
 use chrono::Duration;
 use serde::{Deserialize, Serialize};
-use uuid::Uuid;
 
 use crate::util::ExponentialBackoff;
 
 pub mod client;
 pub mod server;
 
 #[derive(Clone, Debug, Serialize, Deserialize)]
-pub struct IDedMessage<T> {
-    id: Uuid,
+pub struct WrappedMessage<T> {
+    id: Option<String>,
```

Comment:
Minor nit: I'm borderline on this one, but thoughts on creating a type (even if it's just a `String` alias) for the ID?

Reply:
It's less important once we change the signature of `send_response` but that was largely what motivated this suggestion -- something like `RandomID` is more meaningful of a type than `String`.